understanding/20/pause-stop-hide.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Pause, Stop, Hide</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Fewer users are distracted by content that updates or moves</dd>
-		<dt>Author task</dt><dd>Let users control content changes that occur in parallel with other content</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Fewer users are distracted by content that updates or moves.</dd>
+         <dt>What to do</dt><dd>Let users control content changes that occur in parallel with other content.</dd>
+         <dt>Why it's important</dt><dd>Some people with cognitive disabilities and attention deficits cannot concentrate with continual movement.</dd> 
```

Comment:
@GreggVan suggests
```suggestion
         <dt>Why it's important</dt><dd>Some people with cognitive disabilities and attention deficits cannot concentrate with continual movement.  It is also hard to hear screen readers if there is other audio playing at the same time. </dd> 
```

Reply:
The new content is not necessarily audio playing.